core/kernel/dt.c

```diff
@@ -222,7 +222,7 @@ ssize_t _fdt_reg_size(const void *fdt, int offs)
 
 	parent = fdt_parent_offset(fdt, offs);
 	if (parent < 0)
-		return DT_INFO_INVALID_REG;
+		return ((ssize_t)-1);
```

Comment:
for consistency, I suggest to `#define DT_INFO_INVALID_REG_SIZE		((ssize_t)-1)` in **dt.h** and use it here.